<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmInfo</class>
 <widget class="QDialog" name="frmInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Información</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/gr-lida.qrc">
    <normaloff>:/img16/informacion.png</normaloff>:/img16/informacion.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="lb_info_grlida_ico">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/gr-lida.qrc">:/img16/gr-lida.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lb_info_grlida"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lb_info_grlida_version">
     <property name="minimumSize">
      <size>
       <width>46</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>180</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lb_info_qt_ico">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/gr-lida.qrc">:/img16/qt.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lb_info_qt"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lb_info_qt_version">
     <property name="minimumSize">
      <size>
       <width>46</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lb_info_dosbox_ico">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/gr-lida.qrc">:/img16/dosbox.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lb_info_dosbox"/>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="lb_info_dosbox_version">
     <property name="minimumSize">
      <size>
       <width>46</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lb_info_scummvm_ico">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/gr-lida.qrc">:/img16/scummvm.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lb_info_scummvm"/>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lb_info_scummvm_version">
     <property name="minimumSize">
      <size>
       <width>46</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lb_info_vdmsound_ico">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="../res/gr-lida.qrc">:/img16/vdmsound.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="lb_info_vdmsound"/>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="lb_info_vdmsound_version">
     <property name="minimumSize">
      <size>
       <width>46</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lb_info_ico">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>18</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="pixmap">
        <pixmap resource="../res/gr-lida.qrc">:/img16/informacion.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_info">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Información</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QComboBox" name="cbxCategorias">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QTreeWidget" name="twInfo">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Categorías</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <layout class="QHBoxLayout" name="hLayout_1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="hSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/aplicar.png</normaloff>:/img16/aplicar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cbxCategorias</tabstop>
  <tabstop>btnOk</tabstop>
 </tabstops>
 <resources>
  <include location="../res/gr-lida.qrc"/>
 </resources>
 <connections/>
</ui>
